release notes/upcoming.md

```diff
@@ -10,6 +12,7 @@ Description of feature.
 
 ## UX
 * New option to reset cloud token (#672)
+* The option `--no-connection-reuse` now means that keep-alive connections will be disabled. (#676)
```

Comment:
Wouldn't this fit better as a Breaking Change or a Bugfix? Or even if it's in the UX section, a bit more information would probably help, something like:
> The `--no-connection-reuse` option has been re-purposed and now disables keep-alive connections globally. The newly added `--no-vu-connection-reuse` option does what was previously done by `--no-connection-reuse` - it closes any open connections between iterations of a VU, but allows for reusing them inside of a single iteration.

Reply:
I think a breaking change session, it's going to be a better fit.